src/redux/actions/followUnfollowAction.js

```diff
@@ -0,0 +1,88 @@
+import axios from "axios";
+import jwtDecode from "jwt-decode";
+import { toaster } from "evergreen-ui";
+import {
+  FOLLOW_SUCCESS,
+  UNFOLLOW_SUCCESS,
+  NOT_FOLLOWING,
+  FOLLOWING
+} from "../action_types";
+import { serverError } from "./common";
+import config from "../../config";
+
+const baseUrl = config.BASE_URL;
+const token = localStorage.getItem("token");
+
+export const followStatus = author => async dispatch => {
+  const following = () => {
+    dispatch({ type: FOLLOWING });
+  };
+
+  const notFollowing = () => {
+    dispatch({ type: NOT_FOLLOWING });
+  };
+  const allFollowers = await axios.get(
+    `${baseUrl}/profiles/${author}/followers/`,
+    {
+      headers: { Authorization: `Bearer ${token}` }
+    }
+  );
+  const currentUser = jwtDecode(token).username;
+  if (allFollowers.data !== undefined || allFollowers.data.length !== 0) {
```

Comment:
Save the logic in a variable, it's best practice. Such complex logic becomes harder to read esp as an argument within an if statement.

`const hasAllFollowers = allFollowers.data !== undefined || allFollowers.data.length !== 0`

Reply:
@kimotho-njoki Done